### ISSUE TITLE
Error when removing a user from an evaluation

### ISSUE THREAD

**Issue by chvp** [dodona-edu/dodona#4332]
We got a few emails about errors happening during the removal of a student from an evaluation.

Stack trace from an email:
```
Mysql2::Error: Cannot add or update a child row: a foreign key constraint fails (`dodona`.`feedbacks`, CONSTRAINT `fk_rails_be75ac5cac` FOREIGN KEY (`evaluation_user_id`) REFERENCES `evaluation_users` (`id`) ON DELETE RESTRICT ON UPDATE RESTRICT)
  app/controllers/evaluations_controller.rb:139:in `remove_user'
```
Search for request id c7598605-f2de-45d8-949a-d00e2c843f5c in the error emails for more context.

**Comment by jorg-vr** [comment]
I have failed to reproduce this issue.

It has only happened on one occurrence, for one evaluation (With multiple users)

I suggest we ignore it until it happens again

**Comment by bmesuere** [comment]
I also tried without success on mestra

**Comment by bmesuere** [comment]
@jorg-vr I think this can be closed since it was possibly related to #4359

Did we investigate why there were so many duplicates in the database? There's probably a button somewhere we should disable after clicking or so.

**Comment by jorg-vr** [comment]
I am unsure if it is actually related. This issue is caused by a link to feedbacks from evaluation user. (So although both are evaluation related, it are totally different tables)

An evaluation user can have many feebacks and the relation is correctly set to dependent destroy. So it does not look like a similar issue, as the cause in the other pr was that one series should only have one evaluation (While multiple where created)

On the investigation:
I just tested it on naos. The button is not disabled. But clicking three times very fast (with slow internet), results in two requests failing in the background, with the user only noticing the succesful one. (Unless they are looking at their network panel)
![image](https://user-images.githubusercontent.com/21177904/218410000-226f0280-7e79-4380-8c50-e24cdc5bd9f8.png)
So I would not spent much more time on this. This flow will probably receive a rework in #4364 

Conclusion: I would close this issue, and pick it up again if it ever pops up again. Because it only happened once and we cannot reproduce it, it is probably an unlikely edge case and we should not spend to much time on it
